src/js/components/country.js

```diff
@@ -24,7 +24,7 @@
 			var zipcodeWrapper = $( '.form__group--zip' );
 			if ( zipcodeWrapper.length && ( 'US' !== isoCode || 'USA' !== countryName ) ) {
 				zipcodeWrapper.find( 'input' ).val( '00000' );
-				zipcodeWrapper.addClass( 'form__group--active' );
+				zipcodeWrapper.addClass('form__group--active');
```

Comment:
**[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
There must be a space inside this paren. [space-in-parens](https://eslint.org/docs/rules/space-in-parens)